### ISSUE TITLE
Names for call arguments for Java classes should be in same line

### ISSUE THREAD

**Issue by dmitry-oshurkov** [pinterest/ktlint#2663]
## Expected Behavior
```kotlin
Rectangle(
    /* x = */ (x1 * width).roundToInt(),
    /* y = */ (y1 * height).roundToInt(),
    /* width = */ ((x2 - x1) * width).roundToInt(),
    /* height = */ ((y2 - y1) * height).roundToInt()
)

Rectangle(/* x = */ 1040, /* y = */ 321, /* width = */ 81, /* height = */ 96)
```

## Observed Behavior
```kotlin
Rectangle(
      /* x = */
      (x1 * width).roundToInt(),
      /* y = */
      (y1 * height).roundToInt(),
      /* width = */
      ((x2 - x1) * width).roundToInt(),
      /* height = */
      ((y2 - y1) * height).roundToInt()
  )

// A block comment in between other elements on the same line is disallowed (cannot be auto-corrected) (standard:comment-wrapping)
Rectangle(/* x = */ 1040, /* y = */ 321, /* width = */ 81, /* height = */ 96)
```

**Comment by paul-dingemans** [comment]
This behavior is intentional. 

You disable the rule `comment-wrapping` entirely in `.editorconfig` by setting:
```
root = true

[*.{kt,kts}]
ktlint_standard_comment-wrapping = disabled
```

Or suppress statements like below:
```
@Suppress("ktlint:standard:comment-wrapping")
Rectangle(
    /* x = */ (x1 * width).roundToInt(),
    /* y = */ (y1 * height).roundToInt(),
    /* width = */ ((x2 - x1) * width).roundToInt(),
    /* height = */ ((y2 - y1) * height).roundToInt(),
)
```
